scripts/setup-test-data/src/setup_test_data/cli.py

```
@@ -158,46 +158,61 @@ def fetch_releases(client: httpx.Client, repo: str) -> list[Release]:
+            )
+            if len(releases) >= count:
+                break
+    return releases
+
+
+def load_stable_releases(variant: str) -> list[Release]:
+    filepath = RESOURCES_DIR / "releases" / f"{variant_id_pascal(variant)}.json"
+    if not filepath.exists():
+        return []
+
+    with open(filepath, "r") as f:
```

Ruff (0.15.15)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)